hash_table_implementation/hash_table.py

```diff
@@ -0,0 +1,86 @@
+class HashTable(object):
+
+    def __init__(self, size):
+        # Set up size and slots and data
+        self.size = size
+        self.slots = [None] * self.size
+        self.data = [None] * self.size
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpd7804up6/hash_table_implementation/hash_table.py
+++ b/tmp/tmpd7804up6/hash_table_implementation/hash_table.py
@@ -38,6 +38,7 @@
     def hashfunction(self, key, size):
         # Remainder Method
         return key % size
+
     @classmethod
     def rehash(self, oldhash, size):
         # For finding next possible positions
```